<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detector_Choose</class>
 <widget class="QDialog" name="Detector_Choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>检测缺陷选择</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="Cbx_ForeignBody">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>异物检测</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QCheckBox" name="Cbx_WhiteDot">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>白点检测</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="Cbx_LampEye">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>死灯检测</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QCheckBox" name="Cbx_Scratch">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>划伤检测</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="Cbx_boom_light">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>爆灯检测</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QCheckBox" name="Cbx_Abshow_White_Detect">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>移位检测</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="Cbx_mura">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>白印检测</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QCheckBox" name="Cbx_Brightedge">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>亮边检测</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="combx_SelectAllOrNo">
     <property name="currentText">
      <string>全选/全不选</string>
     </property>
     <item>
      <property name="text">
       <string>全选/全不选</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>全选</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>全不选</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn_sure">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn_Cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
